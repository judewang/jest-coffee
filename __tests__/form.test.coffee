post = require '../post'

test 'check for form output', ->
  form = post "http://google.com", { a: 1, b: 2}

  expect(form).toMatchSnapshot()
