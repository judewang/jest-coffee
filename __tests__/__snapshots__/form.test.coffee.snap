// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check for form output 1`] = `
"<form method=\\"post\\" action=http://google.com>
  <input type=\\"hidden\\" name=a value=1>
  <input type=\\"hidden\\" name=b value=2>
  <input type=\\"submit\\" />
</form>"
`;
